esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
Does this switch the code to use dual core?  If so is dual core working ok now, did you test it?